src/server/seeds/development/05_notifications.js

```diff
@@ -19,7 +19,7 @@ exports.seed = function(knex) {
           fk_user_id: 2,
           message: 'You have 5 unread messages',
           has_been_read: false,
-          created_at: '2020-10-01 12:10:00',
+          created_at: '2020-09-01 12:10:00',
```

Comment:
These changes are again not really related to your story. But its better than before